e2e/tests/regression/test_settings.robot

```diff
@@ -17,9 +17,35 @@ Test Setup    Set up test environment
 Test Teardown    Cleanup test resources
 
 *** Test Cases ***
+Test Setting Update With Correct Value
+    [Documentation]    Test multiple settings update and verify the result.
+
+    Update Setting And Assert Value    default-replica-count    1    {"v1":"1","v2":"1"}
```

Comment:
Since we’re validating setting update functionality here, I think separate them into 2 steps:

```
When update setting key to value
Then setting key should be value 
```
will be more clear?

Reply:
Sounds good